package-lock.json

```diff
@@ -12211,7 +12211,7 @@
       "dependencies": {
         "strip-json-comments": {
           "version": "2.0.1",
-          "resolved": "https://registry.npmjs.org/strip-json-comments/-/strip-json-comments-2.0.1.tgz",
+          "resolved": "https://registry.npm.taobao.org/strip-json-comments/download/strip-json-comments-2.0.1.tgz",
```

Comment:
can you rm package-lock from this commit and add it to your git ignore?

Reply:
I have just reverted that commit. Sorry about that..